scripts/linux/agent_diagnostics.sh

```diff
@@ -16,56 +16,70 @@ JCPATH="/opt/jc"
 JCLOG="/var/log/"
 STAMP=$( date +"%Y%m%d%H%M%S" )
 ZIPFILE="./jc${STAMP}.zip"
+TARFILE="./jc${STAMP}.tar"
 declare -a INVENTORY
 
 # Is jcagent installed? If not, exit.
 if [[ ! -d "${JCPATH}" ]]; then
-  echo "JCAGENT IS NOT INSTALLED ON THIS MACHINE."
+  echo "Jcagent is not installed on this machine."
   exit 1
 fi
 
 # Is zip installed?
-if ! which zip 2> /dev/null; then
+if ! which zip &> /dev/null; then
```

Comment:
Why are we redirecting errors to `/dev/null`? I understand not wanting to have standard output display, but do we not want to display errors for troubleshooting purposes?

Reply:
that should have been 1>. tired eyes missed it.